src/data/projects.json

```diff
@@ -736,6 +721,48 @@
       ],
       "repo": "https://github.com/apache/airflow",
       "logo": "airflow-logo.svg"
+    },
+    {
+      "title": "Storybook",
+      "description": "The UI component explorer. Develop, document, & test React, Vue, Angular, Web Components, Ember, Svelte & more!",
+      "language": "TypeScript",
+      "tags":[
+        {
+          "href": "https://discord.gg/storybook",
+          "src": "https://img.shields.io/badge/discord-join-7289DA.svg?logo=discord&longCache=true&style=flat",
+          "alt": "Storybook"
+        }
+      ],
+      "repo": "https://github.com/storybookjs/storybook",
+      "logo": "storybook.png"
+    },
+    {
+      "title": "Babel",
+      "description": "Babel is a compiler for writing next generation JavaScript.",
+      "language": "TypeScript",
+      "tags":[
+        {
+          "href": "https://slack.babeljs.io/",
+          "src": "https://slack.babeljs.io/badge.svg",
+          "alt": "Babel Slack"
+        }
+      ],
+      "repo": "https://github.com/babel/babel",
+      "logo": "babel.png"
+    },
+    {
+      "title": "Jest",
+      "description": "Delightful JavaScript Testing.",
+      "language": "TypeScript",
+      "tags":[
+        {
+          "href": "https://jestjs.io/docs/jest-community",
+          "src": "https://img.shields.io/static/v1?label=Link%20to&message=Community%20Website&color=%3CCOLOR%3E",
+          "alt": "Jest Community"
+        }
+      ],
+      "repo": "https://github.com/facebook/jest",
+      "logo": "facebook.png"
```

Comment:
@Siddhant-K-code I think we should move these projects to the top of the list in descending order based on the number of stars. GitLab is an exception though and we want to keep that at the top of the list no matter what the number of stars are.

Reply:
yeah, thinking about the same. I will do it in my next PR.